test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNet/InstanceAnnotations/AnnotationController.cs

```diff
@@ -0,0 +1,249 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using Microsoft.AspNet.OData;
+using Microsoft.AspNet.OData.Routing;
+using Microsoft.Test.E2E.AspNet.OData.Common.Controllers;
+using NS;
+
+namespace Microsoft.Test.E2E.AspNet.OData.InstanceAnnotations
+{
+    public class EmployeesController : TestODataController
+    {
+        public EmployeesController()
+        {
+            if (null == Employees)
+            {
+                InitEmployees();
+            }
+        }
+
+        /// <summary>
+        /// static so that the data is shared among requests.
+        /// </summary>
+        private static IList<Employee> Employees = null;
+
+        private void InitEmployees()
+        {
+            Employees = new List<Employee>
+            {
+                new Employee()
+                {
+                    ID=1,
+                    Name="Name1",
+                    SkillSet=new List<Skill>{Skill.CSharp,Skill.Sql},
+                    Gender=Gender.Female,
+                    AccessLevel=AccessLevel.Execute,
+                    FavoriteSports=new FavoriteSports()
+                    {
+                        LikeMost=Sport.Pingpong,
+                        Like=new List<Sport>{Sport.Pingpong,Sport.Basketball}
+                    }
```

Comment:
At least there's one "Empoyee" has thee InstanceAnnotations setting.

new Employee()
{
      ....,
      InstanceAnnotations = ...
}

Add a test to send "GET" request to return this employee and verify the instance annotation in the payload.